modules/protocol-tests/resources/META-INF/smithy/test-config.json

```diff
@@ -0,0 +1,1227 @@
+{
+  "smithy": "2.0",
+  "metadata": {
+    "restJsonAllowList": [
```

Comment:
we should adopt the practice of prefixing the metadata in this repo with "alloy"

Reply:
good point